scripts/onnx/test.sh

```diff
@@ -48,6 +48,41 @@ if [[ "$SHARD_NUMBER" == "2" ]]; then
   xdoctest torch.onnx --style=google --options="+IGNORE_WHITESPACE"
 fi
 
+if [[ "$SHARD_NUMBER" == "2" ]]; then
+  # Sanity check on torchbench w/ onnx
+  pip install pandas
+  log_folder="test/.torchbench_logs"
+  device="cpu"
+
+  mkdir -p "${log_folder}"
+  for mode in "accuracy" "performance"; do
+    for compiler in "dynamo-onnx" "torchscript-onnx"; do
+      for suite in "huggingface" "timm_models"; do
```

Comment:
use variables to hold `mode`, `compiler` and `suite` list values? makes it easier to extend later